src/public/indoors/indoor_map_floor.js

```diff
@@ -1,31 +1,51 @@
-var IndoorMapFloor = function(floorId, floorIndex, floorName, floorShortName) {
+var IndoorMapFloor = function (floorId, floorIndex, floorName, floorShortName) {
     var _floorId = floorId;
     var _floorIndex = floorIndex;
     var _floorName = floorName;
     var _floorShortName = floorShortName;
 
-    this.getFloorId = function() {
-        // retain compat with existing API -- id was exposed as short name
-        // whereas it should really be the floorId (a.k.a. z_order)
+    /**
+     * Returns the short name of the floor.
+     *
+     * Retain compat with existing API – id was exposed as short name,
+     * whereas it should really be the floorId (a.k.a. z_order).
+     *
+     * @deprecated use {@link IndoorMapFloor#getFloorShortName} instead.
+     */
+    this.getFloorId = function () {
         return _floorShortName;
     };
-    
-    this._getFloorId = function() {
+
+    /**
+     * Returns the floor id, which matches the z_order in the json submission.
```

Comment:
Returns the z_order of the floor, as defined in the json submission.

Note: this is for compatibility with the existing API - z_order was exposed as id.  The actual id property in the submission json is not accessible through this API.